engine/verification/fetcher/engine.go

```diff
@@ -316,9 +315,7 @@ func (e *Engine) handleValidatedChunkDataPack(ctx context.Context,
 	status *verification.ChunkStatus,
 	chunkDataPack *flow.ChunkDataPack) (bool, error) {
 
-	chunk := status.ExecutionResult.Chunks[status.ChunkIndex]
-	removed := e.pendingChunks.Rem(chunkDataPack.ChunkID)
-
+	removed := e.pendingChunks.Rem(status.ChunkIndex, status.ExecutionResult.ID())
```

Comment:
If it wouldn't be too big of a change, could we rename this method to `Remove` while we're at it 😆

Reply:
`Rem` seems a naming convention for `Remove` method across all of our mempools, e.g., [`ExecutionResults` ](https://github.com/onflow/flow-go/blob/master/module/mempool/results.go#L18). Also, almost all these implementations directly invoke [`Backend.Rem`](https://github.com/onflow/flow-go/blob/1e0a30e7a9118aad96a706639e37ec955f562542/module/mempool/stdmap/results.go#L31-L34). Refactoring `Rem` method for `pendingChunks` would be simple. But refactoring it for all would be a larger surface.